components/authz-service/server/v2/project_update_manager.go

```diff
@@ -390,7 +415,82 @@ func (m *CerealProjectUpdateManager) Status() (ProjectUpdateStatus, error) {
 	return projectUpdateInstance, nil
 }
 
-func (w *workflowInstance) collectAllDomainServiceJobStatuses() []project_update_tags.JobStatus {
+func (w *workflowInstance) collectProgress() (float32, error) {
+	if !w.IsRunning() {
+		return 1, nil
+	}
+	domainServicesUpdateInstance, err := w.GetUpdateDomainServicesInstance()
+	if err != nil {
+		return 0, err
+	}
+
+	progress := []float32{}
+	weight := []int{}
+
+	for _, d := range domainServicesUpdateInstance.ListSubWorkflows() {
+		subWorkflow, err := domainServicesUpdateInstance.GetSubWorkflow(d)
```

Comment:
what's an example of a `subWorkflow`? does it represent something like an ES job on compliance? or is it more specific or more general than that?

Reply:
there are 2 subworkflows: the serialized one and the nodemanager domain service one. both these are allowed to run in parallel